app/src/main/java/org/openobservatory/ooniprobe/model/api/UrlList.java

```diff
@@ -1,9 +0,0 @@
-package org.openobservatory.ooniprobe.model.api;
-
-import org.openobservatory.ooniprobe.model.database.Url;
-
```

Comment:
I couldn't find this class anywhere, where it has been moved?

Reply:
This class was removed on a cleanup of the project, we did this because having unused code will affect the code coverage, and there's no point in covering this with tests if it's not used... There were quite a few cases of Declaration redundancy.

So I ran Analyze->Inspect Code and in Declaration redundancy I removed all unused and redundant classes, methods, etc.... This was one of them.

Reply:
Same as above